stacks_queues/queue.py

```diff
@@ -18,44 +18,49 @@ def __init__(self):
       
 
     def enqueue(self, element):
-        """ Adds an element to the Queue
-            Raises a QueueFullException if all elements
-            In the store are occupied
-            returns None
-        """
-        pass
+        if self.empty():
+            self.front = 0
+            self.rear = 0
+        elif self.rear == self.front:
+            raise QueueFullException("Queue is full!")
+        self.store[self.rear] = element
+        self.rear = (self.rear + 1) % self.buffer_size
 
     def dequeue(self):
-        """ Removes and returns an element from the Queue
-            Raises a QueueEmptyException if 
-            The Queue is empty.
-        """
-        pass
+        if self.front == -1 and self.rear == -1:
```

Comment:
This absolutely works, however as a code style note, it's nice to stay consistent across your methods! In enqueue above, you use `self.empty()` so it would be nice to echo that here. 
```suggestion
        if self.empty():
```